fxtf-drafts/compositing-2/index.html

```diff
@@ -0,0 +1,9 @@
+<!doctype html>
+<head>
+<meta charset="utf-8">
+<title>Moved…</title>
+</head>
+<body onload='(function() { location.href = "https://drafts.csswg.org/compositing-2/" + location.hash; })()'>
+<p>Moved to
+<a href="https://drafts.csswg.org/compositing-1/">https://drafts.csswg.org/compositing-2/</a>
```

Comment:
```suggestion
<a href="https://drafts.csswg.org/compositing-2/">https://drafts.csswg.org/compositing-2/</a>
```

Reply:
I knew that hasty copy-and-paste would not be prove to be such a good idea ;) Thanks!